eva-accession-import-dbsnp2/src/main/java/uk/ac/ebi/eva/accession/dbsnp2/configuration/ImportDbsnpJsonVariantsWriterConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.dbsnp2.configuration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.configuration.annotation.StepScope;
+import org.springframework.batch.item.data.MongoItemWriter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import uk.ac.ebi.eva.accession.core.persistence.DbsnpClusteredVariantEntity;
+
+import static uk.ac.ebi.eva.accession.dbsnp2.configuration.BeanNames.DBSNP_JSON_VARIANT_WRITER;
+
+@Configuration
+public class ImportDbsnpJsonVariantsWriterConfiguration {
+
+    private static final Logger logger = LoggerFactory.getLogger(ImportDbsnpJsonVariantsWriterConfiguration.class);
+
+    @Bean(name = DBSNP_JSON_VARIANT_WRITER)
+    @StepScope
+    public MongoItemWriter<DbsnpClusteredVariantEntity> writer(MongoTemplate mongoTemplate) {
```

Comment:
Agreed offline to move [the DbsnpClusteredVariantWriter class](https://github.com/EBIvariation/eva-accession/blob/master/eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpClusteredVariantWriter.java) to the core submodule so it can be used in this one.

Reply:
As discussed offline, after I'd moved the class to the `eva-accession-core` package, I began to modify it so it doesn't exit with duplicate key errors (`BulkWriteError ... E11000 duplicate key error ..`). Doing so breaks existing tests. (Have added sample code that works for my pipeline (upserts case))